recipes/pybind11_json/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
```

Comment:
```suggestion
conan_basic_setup(TARGETS)

set(CMAKE_CXX_STANDARD 14)
```

Reply:
Conan should learn to work with compile features:
- https://cmake.org/cmake/help/latest/manual/cmake-compile-features.7.html
- https://cmake.org/cmake/help/latest/prop_gbl/CMAKE_CXX_KNOWN_FEATURES.html#prop_gbl:CMAKE_CXX_KNOWN_FEATURES